### PR TITLE
Run code formatter on EEx.Compiler

### PR DESCRIPTION
**Pull request by aleinside**

### CODE REVIEW

File: lib/eex/lib/eex/compiler.ex [hunk 1]
```diff
@@ -34,67 +42,89 @@ defmodule EEx.Compiler do
...
     buffer = insert_quoted(tuples, state.quoted)
     {buffer, rest}
   end
 
   defp generate_buffer([{:end_expr, line, modifier, chars} | t], buffer, [_ | _] = scope, state) do
-    message = "unexpected beginning of EEx tag \"<%#{modifier}\" on end of expression \"<%#{modifier}#{chars}%>\", " <>
-              "please remove \"#{modifier}\" accordingly"
-    :elixir_errors.warn line, state.file, message
+    message =
+      "unexpected beginning of EEx tag \"<%#{modifier}\" on end of expression \"<%#{modifier}#{
```
Comment (josevalim) [inline lib/eex/lib/eex/compiler.ex]: Please change this message so it uses concatenation in different places. For example:

```
"unexpected beginning of EEx tag \"<%#{modifier}\" on end of "  <>
  expression \"<%#{modifier}#{chars}%>\", please remove \"#{modifier}\" accordingly"
```

Or similar.

File: lib/eex/lib/eex/compiler.ex [hunk 2]
```diff
@@ -103,22 +133,31 @@ defmodule EEx.Compiler do
     new_lines = List.duplicate(?\n, line - state.line)
     key = length(state.quoted)
     placeholder = '__EEX__(' ++ Integer.to_charlist(key) ++ ');'
-    {current ++ placeholder ++ new_lines ++ chars,
-     %{state | quoted: [{key, state.engine.handle_end(buffer)} | state.quoted]}}
+
+    {
+      current ++ placeholder ++ new_lines ++ chars,
```
Comment (josevalim) [inline lib/eex/lib/eex/compiler.ex]: Please break the expressions in the tuple out to variables.

File: lib/eex/lib/eex/compiler.ex [hunk 3]
```diff
@@ -34,67 +42,88 @@ defmodule EEx.Compiler do
...
   end
 
   defp generate_buffer([{:start_expr, start_line, mark, chars} | rest], buffer, scope, state) do
     {contents, line, rest} = look_ahead_text(rest, start_line, chars)
+
     {contents, rest} =
-      generate_buffer(rest, state.engine.handle_begin(buffer), [contents | scope],
-                      %{state | quoted: [], line: line, start_line: start_line})
+      generate_buffer(rest, state.engine.handle_begin(buffer), [contents | scope], %{
+        state | quoted: [], line: line, start_line: start_line
```
Comment (whatyouhide) [inline lib/eex/lib/eex/compiler.ex]: Please do this update on its own, like this

```elixir
new_state = %{state | quoted: [], ...}
generate_buffer(rest, state.engine.handle_begin(buffer), [contents | scope], new_state)
```

There's another case where we should do this below :)

**Comment by josevalim** [comment]
:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:

**Pull request merged**